packages/editor/src/client/blocks/richTextBlock.tsx

```diff
@@ -522,6 +710,50 @@ function withRichText<T extends ReactEditor>(editor: T): T {
     }
   }
 
+  editor.deleteBackward = unit => {
+    const {selection} = editor
+
+    if (selection && Range.isCollapsed(selection)) {
+      const [cell] = Editor.nodes(editor, {
+        match: n =>
+          !Editor.isEditor(n) && SlateElement.isElement(n) && n.type === BlockFormat.TableCell
+      })
+
+      if (cell) {
+        const [, cellPath] = cell
```

Comment:
Do you still need that `,` ?

Reply:
Think so, because it is array destructuring. This indicates that the second item is catched. Actually this code comes directly out of slatejs examples.